picard/ui/widgets/scriptdocumentation.py

```diff
@@ -138,7 +138,7 @@ def process_tag(tag: TagVar):
             return f'<dt>{tag_title}</dt><dd>{tag_desc}</dd>'
 
         html = ''
-        for tag in sorted(ALL_TAGS, key=lambda x: x.script_name()):
+        for tag in sorted(ALL_TAGS, key=lambda x: x.script_name().lstrip('_')):
```

Comment:
I think we can have another approach on this, we can expose TagVar.name (instead of having `_name`) which contains exactly what we need here.
It would be simpler and cleaner, see https://github.com/zas/picard/commit/1b59b4bd099dc3837aa048af6f51480743b8e14e

After such changes, this would become
```suggestion
        for tag in sorted(ALL_TAGS, key=lambda x: x.name):
```

What do you think?

Reply:
Actually, I had to use the `_name` attribute instead.

Reply:
You're right.  I missed that.  It definitly makes sense doing what you suggest.  I'll make the changes.  Thanks.